etc/scripts/timetable/import.js

```diff
@@ -193,14 +210,231 @@ var _importCourses = function(ctx, courses, callback) {
     }
 
     var course = courses.pop();
+
     log().info({'id': course.orgUnitId}, 'Importing %s', course.displayName);
-    OrgUnitImport.importOrgUnit(ctx, course.orgUnitId, course, false, function(err) {
+    var options = {
+        'deleteMissing': false,
+        'checkExternalId': false
+    };
+    OrgUnitImport.importOrgUnit(ctx, course.orgUnitId, course, options, function(err) {
         if (err) {
             log().error({'err': err}, 'Failed to import a course');
-            process.exit();
+            process.exit(1);
         }
 
         // Move on to the next course
         _importCourses(ctx, courses, callback);
     });
 };
+
+/**
+ * Export a set of courses. This method will add an `exportedCourse` on each course
+ * holding the entire tree for that course
+ *
+ * @param  {Context}    ctx         Standard context object containing the current user and the current application
+ * @param  {Object[]}   courses     The courses to export
+ * @param  {Function}   callback    Standard callback function
+ * @api private
+ */
+var _exportCourses = function(ctx, courses, callback) {
+    if (_.isEmpty(courses)) {
+        return callback();
+    }
+
+    var course = courses.pop();
+
+    // Export the course so the new ids can be used for the borrowing stage
+    OrgUnitAPI.exportOrgUnit(ctx, course.orgUnitId, 'json', function(err, exportedCourse) {
+        if (err) {
+            log().error({'err': err}, 'Failed to export a course');
+            process.exit(1);
+        }
+
+        course.exportedCourse = exportedCourse;
+
+        // Move on to the next course
+        _exportCourses(ctx, courses, callback);
```

Comment:
This can take a pretty long time, right? Maybe we should log progress information like we do during the export procedure